src/AppBundle/Controller/BlurayController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+
+class BlurayController extends Controller
+{
+    /**
+     * @Route("/Bluray/listing/{page}/{maxperpage}")
+     */
+    public function listingAction($page = 1, $maxperpage = 4)
+    {
+        $listBluray = $this->getDoctrine()->getRepository('AppBundle:Bluray')->findAll();
+        $html = $this->container->get('templating')->render(
+            'bluray/listing.html.twig',
+            array('listBluray' => $listBluray)
+        );
+        return new Response($html);
```

Comment:
You should use http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/view.html to symplify your code.

It's just a recommendation :wink: